### DISCUSSION TITLE
Queue several jobs but run sequentially

### DISCUSSION THREAD

**Discussion by williamoliveira** [timgit/pg-boss#297] [General]
Hello there, is there a way to queue several jobs in a specific queue name and run them sequentially? 
I need to run jobs in different queues in parallel but jobs in the same queue sequentially

**Comment by timgit** [comment]
First of all, this works by design if you are running single-instance.  I think what you're asking is "if I have more than 1 instance, how can I prevent instance 2 from retrieving jobs if instance 1 has retrieved a job but not completed it".  Is that what you mean?

If so, look through the thread in #274 and let me know if you think it's relevant
